src/AST-Core-Tests/RBCodeSnippet.class.st

```diff
@@ -1479,14 +1479,33 @@ RBCodeSnippet class >> styleAllWithError [
 	bigtext inspect
 ]
 
+{ #category : #updating }
+RBCodeSnippet class >> updateAllSnippets [
+	"This script will update all definitions of all snippets.
+	Beware of code loss and double check the generated definitions."
+
+	"I did not manage to update all in a single transformation (with a preview!)
+	So do them one by one with a progress bar :("
```

Comment:
I think you can accumulate changes from all refactorings and preview them with `StRefactoringPreviewPresenter`. Only problem is that presenter requires environment and  scopes. In this case it would make sense to have presenter open with default environment and without scopes.
I could try to refactor it and then we can refactor this to preview changes

Reply:
Thanks! I will improve this part in another PR